_layouts/base.html

```diff
@@ -2,6 +2,7 @@
 layout: default
 ---
 {% include home.html %}
+{% assign keywords = "google,asylo,opensource framework,flexible framework,opensource application,asylo-site,enclave,enclave application" %}
```

Comment:
With this approach, pages cannot override keywords (which seemed to be the intent in the original changes). All pages get exactly these keywords. This is fine, but if this PR is limited to updating the existing global list of keywords, I think I'd prefer just updating the list of keywords where they were instead of creating a variable (that doesn't have a way for it to change) for it.

Reply:
with this PR we can easily update the global lists of keywords, the thing is if we want to use different keywords for different pages like adding a new keyword 'about' for about page, we can do that within the about page and can have a statement in base.html that can assign what keywords we need to choose for the respected pages.

But for now we can have these global lists of keywords that will show on all the pages and if we want some new keywords we can easily add them in the list.

Reply:
> with this commit we can easily update the global lists of keywords

Yes, but that was also true before this change.

> if we want to use different keywords for different pages like adding a new keyword 'about' for about page, we can do that within the about page and can have a statement in base.html that can assign what keywords we need to choose for the respected pages.

This PR doesn't do anything to make it so that the about page can set its own keywords. Like I said before, that's fine, though.

If what this PR accomplishes is updating the hard-coded list of keywords to a new hard-coded list, then I'd prefer just updating the list where it currently is. Moving them up to a variable doesn't match how the other hard-coded values are handled on this page and doesn't allow for anything new in itself.

If the argument is that there should be no meaningful hard-coded values, then that could be a separate conversation and evenly applied throughout.  If the argument is that it will need to be templated in order to plug in content from page front-matter, let's address that if/when we add the ability for the front-matter to control the keywords.